### PR TITLE
Log out from an account

### PR DESCRIPTION
**Pull request by luciademoja**
Added scenario that sees a user logging in and out from an account and display a log out message.

### CODE REVIEW

File: features/step_definitions/user_logout_steps.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Given(/^I am not logged in$/) do
+  visit user_session_path
```
Comment (diraulo) [inline features/step_definitions/user_logout_steps.rb]: @luciademoja you should log out here and not visit the login page.

File: features/user_login.feature [hunk 2]
```diff
@@ -10,7 +10,7 @@ Feature: Login
 
   Scenario: Access the login page
     Given I am on the home page
-    And I click "Login"
+    And I click on "Login"
```
Comment (diraulo) [inline features/user_login.feature:13]: @luciademoja why did you have to change this?

Comment (luciademoja) [inline features/user_login.feature:13]: @diraulo to make it match with a step definition that was already created in another feature

**Comment by diraulo** [comment]
@luciademoja I see you got rid of the step that ensures no one is logged in. Why?

**Pull request closed without merging**